### ISSUE TITLE
Direct messaging users from username in chat

### ISSUE THREAD

**Issue by alanmoo** [mozilla/mattermoz#62]
It should be easier to direct message users, either by
- Double clicking their username in the chat window
- Having a link in the popup that currently appears when clicking on their username 
  ![clementine 2016-03-02_17-08-51](https://cloud.githubusercontent.com/assets/1682681/13513566/bb266a62-e156-11e5-9967-3f2306cab11d.png)

(Or both)

**Comment by xmatthewx** [comment]
+1

**Comment by xmatthewx** [comment]
somewhat related design work: #63

**Comment by alanmoo** [comment]
These are 2 different issues. #63 is about visibility of status, while this is about the ability to click on a username and send a message.

**Comment by xmatthewx** [comment]
You're right. My comment was hasty. I was just linking these issues together so that a designer can take a comprehensive look at everywhere we include username, icon, status, and button to direct message. Editing comment above.
